src/elements/content-sidebar/versions/VersionsSidebar.scss

```diff
@@ -37,3 +37,8 @@
         padding-bottom: 20px;
     }
 }
+
+.upgradeNowButton {
+    color: #0061d5;
```

Comment:
Can we use an existing color variable for this value?

Reply:
+1